components/Stats/index.jsx

```diff
@@ -6,6 +6,7 @@ import { Doughnut } from "react-chartjs-2";
 import flags from "emoji-flags";
 import Container from "../common/Container";
 import { Wrapper, Card, Countries, Flag, ChartWrapper } from "./styles";
+import { LAST_VISITED_COUNTRY } from "../../constants";
```

Comment:
No need to have this in a constants file, it's a single value

Reply:
Yes, but I used it in two files, and I always make typos, so I rather have it as a constant somewhere

Reply:
makes sense